backend/isoko-backend/src/handlers/business/post-list-business.js

```diff
@@ -9,10 +14,79 @@ exports.postListBusinessHandler = async (event) => {
    }
 
    console.info('received:', event);
+   const requestBody = event.body && JSON.parse(event.body);
+   
+   const name = _.get(requestBody, 'name');
+   // add defaultVal param bc city is optional 
+   const city = _.get(requestBody, 'city', "");
+   const type = _.get(requestBody, 'type');
+   const tags = _.get(requestBody, 'tags');
+   const keywords = _.get(requestBody, 'keywords');
+   const shortDesc = _.get(requestBody, 'shortDesc');
+   const businessId = _.get(requestBody, 'businessId');
+   const hours = _.get(requestBody, 'hours');
+   const links = _.get(requestBody, 'links');
+   const address = _.get(requestBody, 'address');
+   const aboutOwner = _.get(requestBody, 'aboutOwner', null);
```

Comment:
If we're gonna set this here, should we set it to an empty object so we can update it later with PUTs?